### DISCUSSION TITLE
Choose specific Query/Mutations to output?

### DISCUSSION THREAD

**Discussion by contextlimit** [MichalLytek/typegraphql-prisma#403] [Q&A]
By default, there's many query/mutations being output that aren't being used, so compilation takes a while. Is it possible to pass in a list of the ones you want output specifically? ie. `output: ['SomeCrudResolver.createManySomething', ...]` or `mutations: ['createManySomething', ...]`

**Comment by MichalLytek** [comment]
https://prisma.typegraphql.com/docs/advanced/exposing-actions

**Comment by contextlimit** [comment]
> https://prisma.typegraphql.com/docs/advanced/exposing-actions

 Oh, thanks! It's not putting the relations on the GraphQL types anymore, do you know how to get those on? I included the relation resolver in the list, ie. FindFirstXResolver generates `type X` and I included `XOnYRelationsResolver` but it doesn't add Y to type X

Edit: nevermind `XRelationsResolver` does it, nice!
